### ISSUE TITLE
A simpler "compact"

### ISSUE THREAD

**Issue by fkereki** [Chalarangelo/30-seconds-of-code#189]
Hi!
The compact() function can be written more simply, by using the `Boolean` function.
```
const compact = (arr) => arr.filter(v => v);
const compact2 = (arr) => arr.filter(Boolean); // simpler

console.log(compact([0, 1, false, 2, '', 3, 'a', 'e'*23, NaN, 's', 34])) // [ 1, 2, 3, 'a', 's', 34 ]
console.log(compact2([0, 1, false, 2, '', 3, 'a', 'e'*23, NaN, 's', 34])) // [ 1, 2, 3, 'a', 's', 34 ]
```

**Comment by Chalarangelo** [comment]
If this doesn't break for any testcases, I'd be glad to merge a PR with this improvement.

**Comment by fkereki** [comment]
Hi!

Thanks for that, will do. Do I require some permission to provide a pull
request?

Best regards,
F.Kereki


On Sat, Dec 16, 2017 at 11:05 AM, Angelos Chalaris <notifications@github.com

**Comment by Chalarangelo** [comment]
Nope, not at all. Just make the changes and send the PR, I'll deal with it later. 👍

**Comment by Chalarangelo** [comment]
Resolved in #207
